browser/BrowserPerf.proto

```diff
@@ -41,7 +41,7 @@ message BrowserPerfData {
     // Page path in the browser, mostly it is URI, without parameter
     string pagePath = 5;
     int32 pagePathId = 6;
-    repeated BrowserErrorLog logs = 7;
+    BrowserErrorLog errorLog = 7;
```

Comment:
Why remove `repeated`? There could be multiple error logs for page loading.

Reply:
When client-js catch an error, it can report log to server at once. if there are multiple errors on client, it can repeatedly report.

Reply:
@Fine0830 Based on `BrowserPerfService#collect`, it includes error logs and perf details. For every page, the perf detail should be reported once, right? And at the same time, BrowserErrorLog could be one or more. 

Then should we separate these two as two report service? Then the clients-js could choose whether to report one log or multiple logs as streaming.

Reply:
Changing the service to
```proto
// Collect performance raw data from the browser.
service BrowserPerfService {
    // report one or more error logs for pages, could report multiple times.
    rpc collectErrorLogs (stream logs) returns (Commands) {
    }
    // report once per page
    rpc collectPerfData (perfDetail) returns (Commands) {
    }
}
```

Reply:
> @Fine0830 Based on `BrowserPerfService#collect`, it includes error logs and perf details. For every page, the perf detail should be reported once, right? And at the same time, BrowserErrorLog could be one or more.
> 
> Then should we separate these two as two report service? Then the clients-js could choose whether to report one log or multiple logs as streaming.

Generally, when user open the page, perf data is produced. Error logs are produced in many case, like loading, click, user interactions.  Now, there is a list to store error logs and perf logs in client-js, then the list item will report logs to server in turn.  Another way, we can regularly report  all logs in the list.

Reply:
@wu-sheng you mean we separate these two as error and perf report service? If that, it is OK.

Reply:
Yes. From your description, It seems more reasonable

Reply:
I'll change the proto.

Reply:
@arugal From https://github.com/apache/skywalking-query-protocol/pull/30#issuecomment-581715189, I think this comment also supports this change. Only the error log should be persistent and provide query.